README.md

```diff
@@ -365,6 +365,112 @@ You can also specify the delimiter (`comma`) for the CSV format (default: `,`):
 }
 ```
 
+## Preprocessing responses
+
+Responses in other formats than JSON can be preprocessed by calling any command line tool that can produce JSON output. The response body is piped to the `stdin` of the tool and the result is read from `stdout`. The result of the command is then used as the actual response and is checked. This only works, if the command line tool produces valid JSON.
+
```

Comment:
I would like to be able to process all responses, no matter of the format. E.g. we want to be able to "strip" metadata from a say JPEG file and compare the JPEG binary data to an expected response.

So 

```
"response": {
  "format": {
    "pre_process": {
       ... strip metadata ...
    },
    "type": "binary"
}
```

should be supported.

Reply:
This was already supported and documented right below this paragraph, including an example with exiftool. 

I added another example where exiftool produces xml which is then formatted to json.